### ISSUE TITLE
Non interactive

### ISSUE THREAD

**Issue by pemontto** [w33ble/kbn-shoehorn#1]
Really appreciate this, it saves me manually doing it or scripting it up myself. I was wondering if I could just supply the answers as command line arguments?

**Comment by w33ble** [comment]
Glad you enjoy it. You can pass in the repo path already, it's just not mentioned in the readme (should probably update that). For example, `kbn-shoehorn sivasamyk/logtrail` will work, and skip asking you about the download path.

Adding a non-interactive way to use this is a great idea. I think the branch is the only part you might need to include, the questions about changing the version and fixing the path are probably always going to be yes anyway.

What do you think about 2 new flags, one to just default to yes (say, perhaps, `-y`), and another to allow you to pass the branch? My thinking is that if you pass `-y`, it'll assume the master branch, but you can override if with another flag (probably `--branch`). So using it would just be:

```
kbn-shoehorn sivasamyk/logtrail -y

# or with a custom branch

kbn-shoehorn sivasamyk/logtrail -y --branch=5.4.0
```

**Comment by w33ble** [comment]
Sorry I didn't wait for a response, I just went ahead and implemented it the way I said.

Just published v0.2.0 of the package on npm, which now has that functionality. Thanks for the idea!

**Comment by pemontto** [comment]
Completely agreed anyway, thanks @w33ble!
